app/assets/javascripts/rdc_angular/controllers/experience_stage.js

```diff
@@ -2,11 +2,20 @@ app.controller('ExperienceStageCtrl', function($scope, $document, $timeout) {
   var hash = window.location.hash;
   $scope.show_all = hash != '';
 
+  // On page load, we initialize theme_:id_experiences=false, unless that theme is listed in the url hash.
+  $scope.initialize = function(themeIdExperiences) {
+    // Use the hash to generate the scope variable that we'd like to set to `true`.
+    // For example, with `hash`, of the form `#theme-1` => construct scope variable name `theme_1_experiences`
+    var themeIdExperienceToShow = (hash.split('#')[1]+'_experiences').replace(/-/,'_');
+
+    $scope[themeIdExperiences] = (themeIdExperiences === themeIdExperienceToShow) ? true : false;
```

Comment:
Though this is hacky, I think this was a good way to work around what we needed – good work!